Sources/FoundationEssentials/AttributedString/Conversion.swift

```diff
@@ -42,13 +37,13 @@ extension String {
         // the original AttributedString release.
         self.init(characters[...])
     }
+    #endif
 
     @available(macOS 9999, iOS 9999, tvOS 9999, watchOS 9999, *)
     @usableFromInline
     internal init(_characters: AttributedString.CharacterView) {
```

Comment:
Was this intentionally moved outside of the `#if` conditional?

Reply:
Yes! It's for use within project internals, and to establish the ABI entry point, whether or not we manage to make a public wrapper for it.